apps/data-backend/src/routes/indexer/all-transactions.ts

```diff
@@ -0,0 +1,52 @@
+import type { FastifyInstance, RouteOptions } from "fastify";
+import { prisma } from "indexer-prisma";
+import { HTTPStatus } from "../../utils/http";
+import { isValidStarknetAddress } from "../../utils/starknet";
+
+interface AllTransactionsParams {
+  tokenAddress: string;
+}
+
+async function allTransactionsRoute(
+  fastify: FastifyInstance,
+  options: RouteOptions
+) {
+  fastify.get<{
+    Params: AllTransactionsParams;
+  }>("/all-transactions/:tokenAddress", async (request, reply) => {
+    try {
+      const { tokenAddress } = request.params;
+      if (!isValidStarknetAddress(tokenAddress)) {
+        reply.status(HTTPStatus.BadRequest).send({
+          code: HTTPStatus.BadRequest,
+          message: "Invalid token address"
+        });
+        return;
+      }
+
+      const allTransactions = await prisma.token_transactions.findMany({
+        where: {
+          memecoin_address: tokenAddress
+        },
+        select: {
+          memecoin_address: true,
+          price: true,
+          liquidityRaised: true,
+          total_supply: true,
+          network: true
```

Comment:
we need to also display: 
- transaction_type: buy and sell
- amount: send of the base token (ETH/STRK paid or receive when sell the token)
- the amount received: coin_received or sell
- created_at
- owner address